### ISSUE TITLE
Let the module developer specify fields he's interested in

### ISSUE THREAD

**Issue by mokaddem** [MISP/misp-modules#287]
Let the module developer specify fields he's interested in as a kind of filter rule

**Comment by stricaud** [comment]
right, so to clarify this point here, I am writing an export module, and only need a few fields as input, not all of them. If I could filter this out, that would save me getting data I don't need.
